pkg/auth/dependency/newinteraction/nodes/utils_authn.go

```diff
@@ -0,0 +1,33 @@
+package nodes
+
+import (
+	"github.com/authgear/authgear-server/pkg/auth/dependency/authenticator"
+	"github.com/authgear/authgear-server/pkg/auth/dependency/newinteraction"
+	"github.com/authgear/authgear-server/pkg/core/authn"
+)
+
+func getAuthenticators(ctx *newinteraction.Context, graph *newinteraction.Graph, stage newinteraction.AuthenticationStage, typ authn.AuthenticatorType) ([]*authenticator.Info, error) {
```

Comment:
Add `TODO(new_interaction): Support using owned authenticators not allowed in config`